src/interfaces/IBLSSignatureChecker.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity =0.8.12;
 
 import {IRegistryCoordinator} from "src/interfaces/IRegistryCoordinator.sol";
 import {IBLSApkRegistry} from "src/interfaces/IBLSApkRegistry.sol";
-import {IStakeRegistry, IDelegationManager, IServiceManager} from "src/interfaces/IStakeRegistry.sol";
+import {IStakeRegistry, IDelegationManager} from "src/interfaces/IStakeRegistry.sol";
```

Comment:
Nit: The natspec for the event `StaleStakesForbiddenUpdate` notes that the value is set by the serviceManagerOwner

Reply:
[da8f669](https://github.com/Layr-Labs/eigenlayer-middleware/pull/98/commits/da8f669aa139569306a427c1eac0e9410a5852ce)